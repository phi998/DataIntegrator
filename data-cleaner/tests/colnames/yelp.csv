Name,Name,Rating,Rating,Opening Hours,Number of Photos,Address,Location,Location,Phone Number,Address,Currency,Features,Features,OtherName,Name,Rating,Rating,Opening hours,Number of Photos,Address,Location,Location,Phone Number,Address,other,Feature,Location,Other
Name,Name,Rating,Rating,Opening Hours,Actions,Address,Location,Location,Phone Number,Address,Rating,Actions,Labels or actions,Additional informations
Name,Name,Rating,Rating,Opening Hours,Labels/actions,Address,Location,Location,Additional informations,Address,Rating,Labels/actions,Labels/actions,Additional informations
