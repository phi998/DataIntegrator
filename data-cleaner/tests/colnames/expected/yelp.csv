Name,Name,Rating,Rating,Opening Hours,Labels/actions,Address,Location,Location,Phone number,Address,Price,Labels/actions,Labels/actions,Additional informations
